.github/workflows/main.yaml

```diff
@@ -0,0 +1,48 @@
+name: Main
+
+on:
+    push:
+        branches:
+            - main
+            - master
+    pull_request:
+        branches:
+            - main
+            - master
```

Comment:
This repository uses the branch "master" so you can drop the "main" from those lists